src/mds/Locker.cc

```diff
@@ -2737,7 +2737,7 @@ void Locker::handle_client_caps(MClientCaps *m)
       mdcache->wait_replay_cap_reconnect(m->get_ino(), new C_MDS_RetryMessage(mds, m));
       return;
     }
-    dout(1) << "handle_client_caps on unknown ino " << m->get_ino() << ", dropping" << dendl;
+    dout(7) << "handle_client_caps on unknown ino " << m->get_ino() << ", dropping" << dendl;
```

Comment:
Also needs to be a separate PR and the commit needs amended to note it fixes the issue. Sorry to be a PITA Zheng but this makes backporting easier and helps me get these fixes expedited.